packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -878,37 +878,37 @@ export class AttributesController extends ConsumptionBaseController {
         await this.attributes.delete({ id: id });
     }
 
-    public async deleteRepositoryAttribute(sourceAttribute: LocalAttribute): Promise<void> {
-        const validationResult = await this.validateSourceAttributeDeletion(sourceAttribute);
+    public async executeFullAttributeDeletionProcess(attribute: LocalAttribute): Promise<void> {
```

Comment:
1. is there a "not full" attribute deletion process?
2. why don't we simply call it deleteAttribute? I think we can abstract away from the user that there is triggered a "process".

Reply:
We already have a deleteAttribute function, which "only" deletes the Attribute with respective child Attributes (and fires an event). In contrast, this "full process" also deletes the predecessors, detaches all respective shared versions, and detaches the successor.